generators/client-2/templates/src/main/webapp/app/components/_common.module.ts

```diff
@@ -15,7 +15,7 @@ import { AuthServerProvider } from './auth/auth-jwt.service';
 import { AuthServerProvider } from './auth/auth-session.service';
 <%_ } _%>
 import { Account } from './auth/account.service';
-import { LoginService } from './login/login.service';
+//import { LoginService } from './login/login.service';
```

Comment:
same, remove commented lines in this file as we import the login service in appmodule now.